addons/sale_timesheet/models/project_update.py

```diff
@@ -21,12 +21,15 @@ def _get_template_values(self, project):
             'show_activities': template_values['show_activities'] or show_sold or bool(profitability),
             'services': services,
             'profitability': profitability,
+            'format_value': lambda value, is_hour: str(round(value, 2)) if not is_hour\
+                            else "%s:%s" % (str(int(value)).rjust(2, '0'), str(int(value * 60) % 60).rjust(2, '0')),
```

Comment:
Couldn't we use `format_duration` from `odoo.tools.misc` to format the time in case is_hour is true?

Reply:
true that, thanks :fire: